Gopkg.toml

```diff
@@ -44,23 +44,23 @@
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.15.4"
+  version = "kubernetes-1.16.3"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.15.4"
+  version = "kubernetes-1.16.3"
 
-[[override]]
+[[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.15.4"
+  version = "kubernetes-1.16.3"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.15.4"
+  version = "kubernetes-1.16.3"
 
 [[constraint]]
   name = "sigs.k8s.io/controller-runtime"
-  version = "v0.3.0"
+  version = "v0.4.0"
```

Comment:
Is this strictly needed? I haven't looked deeper yet, but Tim pinned `operatorkit` to `0.3.0`. Is there something special with `v0.4.0`? Distinction between pre- and post-k8s 1.16?

Reply:
If I understand well, 0.4.0 is first one with k8s 1.16 support https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.4.0